content/guides/quickstart/groups-guide.md

```diff
@@ -0,0 +1,1535 @@
+          ?:  (gte h.tarp 1)
+            ;/  "{(a-co:co h.tarp)} hours ago"
+          ;/  "{(a-co:co m.tarp)} minutes ago"
+        =/  =tarp  (yell (sub d now.bol))
+        ?:  (gte d.tarp 1)
+          ;/  "{(a-co:co d.tarp)} days"
+        ?:  (gte h.tarp 1)
+          ;/  "{(a-co:co h.tarp)} hours"
+        ;/  "{(a-co:co m.tarp)} minutes"
+  ==
+::
+++  style
+  ^~
+  ^-  tape
+  %-  trip
+  '''
+  * {font-family: monospace}
+  h3 {display: inline}
+  table {margin: 1em}
+  th {text-align: right; vertical-align: middle;}
+  td {padding-left: 1em; vertical-align: middle;}
+  td form {margin: 0}
+  label {
+    display: inline-block;
+    margin-right: 1em;
+    min-width: 9ch;
+    vertical-align: middle;
+  }
+  select {min-width: 8ch}
+  #s, #u {margin-left: 1ch}
+  #submit {margin-top: 1em}
+  #yea {margin-right: 1ch}
+  '''
+--
+```
+
+### Desk config
+
+With our types, agent, mark files and front-end now complete, the last thing we
+need are some desk configuration files.
+
+Firstly, we need to specify the kernel version our app is compatible with. We do
+this by adding a `sys.kelvin` file to the root of our `squad` directory:
+
+```shell {% copy=true %}
+cd squad
+echo "[%zuse 418]" > sys.kelvin
+```
+
+We also need to specify which agents to start when our desk is installed. We do
+this in a `desk.bill` file:
+
+```shell {% copy=true %}
+echo "~[%squad]" > desk.bill
+```
+
+Lastly, we need to create a Docket file. Docket is the agent that manages app
+front-ends - it fetches & serves them, and it also configures the app tile and
+other metadata. Create a `desk.docket-0` file in our `squad` working directory
+and add the following:
+
+```hoon {% copy=true %}
+:~
+  title+'Squad'
+  info+'A simple groups app.'
+  color+0x4b.c647
+  version+[0 1 0]
+  website+'https://urbit.org'
+  license+'MIT'
+  base+'squad'
+  site+/squad
+==
+```
+
+### Put it together
+
+Our app is now complete, so let's try it out. In the Dojo of our comet,
+we'll create a new desk by forking from an existing one:
+
+``` {% copy=true %}
+|merge %squad our %webterm
+```
+
+Next, we'll mount the desk so we can access it from the host OS:
+
+``` {% copy=true %}
+|mount %squad
+```
+
+Currently its contents are the same as the `%webterm` desk, so we'll need to
+delete those files and copy in our own instead. In the normal shell, do the
+following:
+
+```shell {% copy=true %}
+rm -r dev-comet/squad/*
+cp -r squad/* dev-comet/squad/
+```
+
+Back in the Dojo again, we can now commit those files and install the app:
+
+``` {% copy=true %}
+|commit %squad
+|install our %squad
+```
+
+If we open a web browser, go to `localhost:8080`, and login with the password
+obtained by running `+code` in the Dojo, we should see a tile for the Tally app.
+If we click on it, it'll open our front-end and we can start using it.
+
+One thing we can also do is publish the app so others can install it from us. To
+do so, just run the following command:
+
```

Comment:
May be worth summarizing in these here:  "you should have the following files present"

Reply:
sorry which files do you mean to summarise here? The contents of the desk?

Reply:
That's what I mean, yes.